vm/chef/cookbooks/solr/attributes/default.rb

```diff
@@ -12,5 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-default['solr']['version'] = '8.6.2'
+require 'net/http'
+
+# Find latest version for Solr 8
+uri = URI('https://downloads.apache.org/lucene/solr/')
```

Comment:
Shouldn't we be keeping these builds hermetic instead?

Looks like this directory has older versions
https://archive.apache.org/dist/lucene/solr/

Reply:
👍🏻  I prefer going for your solution.